code/alldvds.html

```diff
@@ -0,0 +1,262 @@
+        </div>
+        <div class="dropdown-content">
+            <div class="dropdown-small">
+                <div>
+                    <button class="droptxt-small">
+                        <a href="index.html">HOME</a>
+                    </button>
+                </div>
+                <div class="dropdown-content-small">
+                </div>
+            </div>
+            <div class="dropdown-small">
+                <div>
+                    <button class="droptxt-small">
+                        <a href="alldvds.html">DVD-LIBRARY</a>
+                    </button>
+                </div>
+                    <div class="dropdown-content-small">
+                        <a href="alldvds.html">ALL DVDs</a>
+                        <a href="comedy.html">COMEDY</a>
+                        <a href="action.html">ACTION</a>
+                        <a href="romance.html">ROMANCE</a>
+                        <a href="horror.html">HORROR</a>
+                        <a href="other.html">OTHER</a>
+                    </div>
+            </div>
+            <div class="dropdown-small">
+                <div>
+                    <button class="droptxt-small">
+                        <a href="register.html">SIGN UP</a>
+                    </button>
+                </div>
+                    <div class="dropdown-content-small">
+                        <a href="alldvds.html">ALL DVDs</a>
+                        <a href="comedy.html">COMEDY</a>
+                    </div>
+            </div>
+            <div class="dropdown-small">
+                <div>
+                    <button class="droptxt-small">
+                        <a href="cart.html"><img src="shopping-cart-small.png"></a>
+                    </button>   
+                </div>
+                <div class="dropdown-content-small">
+                </div>
+            </div>
+        </div>
+    </div>
+
+  </header>
+
+  <section class="intro">
+    <section class="section-header">
+      <img src="dvd.png" alt="img or icon">
+      <div class="header-title">
+        <h1>Watching movies to improve your language skills!</h1>
+      </div>
+    </section>
+
+  <section>
+    <!-- what's new, List of recently added dvds-->
+    <h3>ALL AVAILABLE DVDs</h3>
+    <div class="DVD-card-holder">
+      <div class="DVD-card">
+        <div class="DVD-title">
+          <div class="DVD-title-item">Nachts im Museum</div>
+        </div>
+        <img src="./images/nachtsim.jpg" alt="DVD-cover">
+        <div class="description">
+          Tagträumer Larry beginnt als Nachtwächter im Naturhistorischen
+          Museum. Doch statt eines gemütlichen Jobs erwartet ihn die grösste
+          Herausforderung seines Lebens. Die Exponate, darunter ausgestorbene
+          Tiere, Neandertaler, Gladiatoren, Cowboys und das gigantische Skelett
+          eines T-Rex, erwachen zum Leben und machen sich auf, das Museum zu
+          verwüsten.
+        </div>
```

Comment:
Some of the descriptions are taking up more space than the images, which makes the card jump (grow) a little bit on hover. Try setting a fixed height on your `DVD-card`s to prevent this